.github/workflows/build-on-windows.yml

```
@@ -0,0 +1,34 @@
+name: Test build on Windows
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/build-on-windows.yml'
+      - RNReanimated.podspec
+      - android/build.gradle
+      - package.json
+  push:
+    branches:
+      - main
+
+env:
+  REACT_NATIVE_VERSION: "0.70.5"
+
+jobs:
+  build:
+    runs-on: windows-latest
+    concurrency:
+      group: build-on-windows-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+      - name: Create React Native app
+        run: npx react-native init app --version ${{ env.REACT_NATIVE_VERSION }}
```

Nothing important, but there's a convention that we name all apps with PascalCase, i.e. `Example`, `FabricExample`, `WebExample`, so let's use `App` or `MyApp` here 😄 First I was thinking of `WindowsExample` but this is an Android app, just built in Windows OS.
```suggestion
        run: npx react-native init MyApp --version ${{ env.REACT_NATIVE_VERSION }}
```